docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -185,61 +181,87 @@ repo_gpgcheck=1
+`/opt/openziti/etc/identities`. Add identity files to this directory by copying the identity file into the directory and
+setting permissions for group `ziti` so the tunneler service may delete JWT enrollment tokens and read and write the
+JSON identity configuration files.
+
+```text
+sudo chown -cR :ziti        /opt/openziti/etc/identities
+sudo chmod -cR ug=rwX,o-rwx /opt/openziti/etc/identities
+```
+
+The tunneler process needs to be restarted if the contents of `/opt/openziti/etc/identities` change.
+
+```text
+sudo systemctl restart ziti-edge-tunnel.service
+```
+
+### Load a Single Identity Without Restarting
+
+This sends an IPC command to the running tunneler and requires write permission on the IPC socket
+(`/tmp/.ziti/ziti-edge-tunnel.sock`). Run this command as the logged in user instead of user `ziti` if a member of group
+`ziti`.
+
+```text
+sudo -u ziti \
+  ziti-edge-tunnel add --jwt "$(< ./in-file.jwt)" --identity myIdentityName
+```
 
 ## Manual Installation
 
-[The latest binary release](https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/) of `ziti-edge-tunnel` is distributed as an executable for amd64, arm, arm64 architectures. The upgrade procedure is identical to the installation procedure.
+[The latest binary release](https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/) of `ziti-edge-tunnel` is
+distributed as an executable for amd64, arm, arm64 architectures. The upgrade procedure is identical to the installation
+procedure.
 
-You'll need to install the `wget` and `unzip` commands to use this example.
+Install the `wget` and `unzip` commands to use this example.
 
 ```text
-wget -q "https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/download/ziti-edge-tunnel-Linux_$(uname -p).zip" \
+wget -q \
+  "https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/download/ziti-edge-tunnel-Linux_$(uname -p).zip" \
   && unzip ./ziti-edge-tunnel-Linux_$(uname -p).zip \
   && rm ./ziti-edge-tunnel-Linux_$(uname -p).zip \
   && chmod -c +x ./ziti-edge-tunnel \
   && ./ziti-edge-tunnel version
 ```
 
-### Enroll Before You Run
+### Enroll Before Run
 
-You will need the token file or its contents to enroll. Enrollment is the act of exchanging the token for an identity that is to be permanently installed in the filesystem.
+A one-time token (JWT) is required to enroll.
+
+```text
+ziti-edge-tunnel enroll --jwt ./in-file.jwt --identity ./out-file.json
+```
 
-[Learn more about enrolling](/docs/learn/core-concepts/identities/enrolling).
+[Learn more about enrolling](/learn/core-concepts/identities/20-enrolling.md).
 
 ### Run the Manually Installed Binary
 
 ```text
-ziti-edge-tunnel run \
-  --identity-dir /opt/openziti/etc/identities
+ziti-edge-tunnel run --identity-dir /opt/openziti/etc/identities
 ```
 
-[Learn more about tunneler options and modes](./linux-tunnel-options).
+[Learn more about tunneler options and modes](./linux-tunnel-options.md).
```

Comment:
is manual mode the only time one would/should learn more about tunneler options and modes? this is the only place is appears (under manual)